Gemfile

```diff
@@ -17,7 +17,9 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby] # Windows doe
 # gem 'bcrypt' # Use Active Model has_secure_password
 
 # Authentications & Authorizations
+gem 'devise' # Authentication for Ruby Application
 gem 'pundit' # Minimal authorization through OO design and pure Ruby classes
+gem 'doorkeeper' # OAuth2 for enabling authentication on Mobile Applications
```

Comment:
@aetheryna Do we need this now or we make add it later when we have to? 💭

Reply:
@Lahphim We don't really need it now now that I think about it 🤔 , how about I comment it out or would you prefer it if I add this when I work on the Mobile Application? 😮